react/features/conference/components/native/NavigationBar.js

```diff
@@ -99,7 +99,8 @@ function _mapStateToProps(state) {
         _conferenceTimerEnabled:
             getFeatureFlag(state, CONFERENCE_TIMER_ENABLED, true) && !state['features/base/config'].hideConferenceTimer,
         _meetingName: getConferenceName(state),
-        _meetingNameEnabled: getFeatureFlag(state, MEETING_NAME_ENABLED, true),
+        _meetingNameEnabled:
+            getFeatureFlag(state, MEETING_NAME_ENABLED, true) && !state['features/base/config'].hideConferenceSubject,
```

Comment:
can you please extract all variables from the same state at the top?

Reply:
good call, done